databases/week2/DB_W2_HW2.sql

```diff
@@ -0,0 +1,42 @@
+-- Create a new database containing the following tables
+CREATE DATABASE school;
+
+USE school;
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+DROP TABLE class;
+CREATE TABLE class 
+(id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+name VARCHAR(50),
+begin_date DATETIME NOT NULL,
+end_date DATETIME,
+PRIMARY KEY (id));
+
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE TABLE student 
+(id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+name VARCHAR(50) NOT NULL,
+email VARCHAR(50),
+phone INTEGER(8),
+class_id INT UNSIGNED,
+PRIMARY KEY (id),
+FOREIGN KEY (class_id) REFERENCES class(id));
```

Comment:
This is great. Again my only suggestion is indenting the column definitions for each table. Something like this:

```sql
CREATE TABLE student (
    id INT UNSIGNED NOT NULL AUTO_INCREMENT,
    name VARCHAR(50) NOT NULL,
    email VARCHAR(50),
    phone INTEGER(8),
    class_id INT UNSIGNED,
    PRIMARY KEY (id),
    FOREIGN KEY (class_id) REFERENCES class(id)
);
```

Reply:
Sure. Thanks